packages/web/src/components/organisms/TriggersFormItems/ActionFormItems.tsx

```diff
@@ -14,14 +14,21 @@ import TriggerSelectorSwitcher from './TriggerSelectorSwitcher';
 const ActionFormItems = () => {
   const {keyMap} = useTriggersPick('keyMap');
 
+  const [actionValue, setActionValue] = useState<string>('');
```

Comment:
You don't need to store `actionValue` in state, instead you may use `Form.useWatch`:
```suggestion
  const actionValue = Form.useWatch('action');
```
or optionally `form.getFieldValue('action')`